app/models/event.rb

```diff
@@ -72,20 +74,26 @@ def meetup_url meetup_event_id
     "http://#{meetup_group_url}/events/#{meetup_event_id}/"
   end
 
-  def at_limit?
+  def students_at_limit?
     if student_rsvp_limit
       student_rsvps_count >= student_rsvp_limit
     end
   end
 
+  def volunteers_at_limit?
+    if volunteer_rsvp_limit
+      volunteer_rsvps_count >= volunteer_rsvp_limit
+    end
+  end
+
   def survey_sent?
     !!survey_sent_at
   end
 
   def validate_rsvp_limit
-    return unless persisted? && student_rsvp_limit
+    return unless persisted? && student_rsvp_limit || persisted? && volunteer_rsvp_limit
 
-    if student_rsvp_limit < student_rsvps_count
+    if (student_rsvp_limit < student_rsvps_count) || (volunteer_rsvp_limit < volunteer_rsvps_count)
```

Comment:
These should probably be added as two separate errors (for students & volunteers)